packages/twenty-front/src/modules/client-config/graphql/queries/getClientConfig.ts

```diff
@@ -29,7 +29,7 @@ export const GET_CLIENT_CONFIG = gql`
       defaultSubdomain
       frontDomain
       debugMode
-      analyticsEnabled
+      isAnalyticsEnabled
```

Comment:
Maybe let's rename the env variable only since this will create a breaking change during the next deploy :(